### PR TITLE
Branch change delete

### PR DESCRIPTION
**Pull request by imrajsinghsandhu**
Change Delete command to allow for deletion of Application via Job Applicant's name

Edit execute method of DeleteCommand to allow for deletion of a user from the list, via name
Add method isStringInt that checks if the input is an Integer, in DeleteCommandParser
Make changes to tests to allow for deletion via name

### CODE REVIEW

**Review by eman-kom — COMMENTED** [review]
Overall looks good 👍 
Could you also remove all the binary files?

**Comment by imrajsinghsandhu** [comment]
> Overall looks good 👍 Could you also remove all the binary files?

Yes, have done so. Thanks for pointing it out!

File: src/main/java/seedu/address/model/person/Name.java [hunk 1]
```diff
@@ -38,6 +38,10 @@ public static boolean isValidName(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    public String getfullName() {
```
Comment (rcjj98) [inline src/main/java/seedu/address/model/person/Name.java:41]: Should be in camelCase: getFullName()

**Review by rcjj98 — CHANGES_REQUESTED** [review]
1 nitpick change requested! 
Other than that LGTM 🤩

File: src/main/java/seedu/address/model/person/Name.java (hunk 1 above)
Comment (imrajsinghsandhu) [reply src/main/java/seedu/address/model/person/Name.java:41]: Right, thanks for pointing it out! Will edit it.

**Review by rcjj98 — APPROVED** [review]

**Review by brigittesantoso — COMMENTED** [review]
LGTM

**Pull request merged**